frontend/src/features/admin-form/settings/components/PaymentSettingsSection/PaymentSettingsSection.tsx

```diff
@@ -40,28 +43,54 @@ import {
 } from './StripeConnectButton'
 
 const PaymentsDisabledRationaleText = ({
-  isEmailsPresent,
+  isAdminEmailsPresent,
   isSingleSubmission,
+  isPDFResponseEnabled,
 }: {
-  isEmailsPresent: boolean
+  isAdminEmailsPresent: boolean
   isSingleSubmission: boolean
+  isPDFResponseEnabled: boolean
 }): JSX.Element => {
-  if (isEmailsPresent && isSingleSubmission) {
+  const disabledCount = [
+    isAdminEmailsPresent,
+    isSingleSubmission,
+    isPDFResponseEnabled,
+  ].filter(Boolean).length
+
+  const { data: formDef } = useAdminForm()
+
```

Comment:
Would prefer to do a guard check for `formDef` if it is not available.

```
if(!formDef) return null;
```

Reply:
86e23b9a92d38534a58ca50fd95a45b685aaa642

Fixed!